lib/generators/boring/database_cleaner/active_record/rspec/install/install_generator.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+module Boring
+  module DatabaseCleaner
+    module ActiveRecord
+      module Rspec
+        class InstallGenerator < Rails::Generators::Base
+          desc 'Adds database_cleaner-active_record gem to the application for cleaning ActiveRecord databases using different strategies'
```

Comment:
Let's update to "Adds database_cleaner-active_record gem to the application for cleaning databases during tests"